Firebase.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = 'Firebase'
```

Comment:
@paulb777, two outstanding questions:
1. Does this PR need a CHANGELOG entry for the updated min. versions for products like GTMSessionFetcher, GDT, and GULs? (My guess is no.)
2. Does the ZipBuilder need to be updated to build the latest GoogleSignIn and/or AdMob? (I assume the ZipBuilder already does so no action is needed.)

Reply:
1. Yes for GTMSessionFetcher since that is supported for non-Google usages.
2. Should be automatic, so no.

Also, I just remembered https://github.com/firebase/firebase-ios-sdk/blob/master/FirebaseCore/Tests/Unit/FIROptionsTest.m#L629

Reply:
Got it. Also, I forget the procedure. Do I need to stage these specs for CI to pass?